demos/dashboards-investor-preferences/demo.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="demo.css" />
+        <script src="https://code.highcharts.com/dashboards/datagrid.src.js"></script>
+        <script src="https://code.highcharts.com/highcharts.src.js"></script>
+        <script src="https://code.highcharts.com/dashboards/dashboards.src.js"></script>
+        <script src="../../code/connectors-morningstar.src.js"></script>
+
+        <title>Highcharts Dashboards + Morningstar Portfolio Investment Details</title>
+    </head>
+    <body>
+        <h1>Highcharts Dashboards + Morningstar Portfolio Investment Details</h1>
+        <p>
+            Add your Postman environment file from Morningstar to start the demo:
+            <input type="file" id="postman-json" accept=".json,application/json" />
+        </p>
+        <p id="loading-label" style="display: none;">Loading data…</p>
+        <div class="filters-row">
```

Comment:
1.Let's place filter in a row, one next to each other, not vertically:
![image](https://github.com/user-attachments/assets/da95a022-7d78-416b-a10b-b119d607622e)

2. Let's disable (hide?) filters until user provides the env file